src/frontend/src/components/public/PIASubHeader/index.tsx

```diff
@@ -0,0 +1,122 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faEllipsisH } from '@fortawesome/free-solid-svg-icons';
+
+import { statusList } from '../../../utils/status';
+import { PIASubHeaderProps } from './interfaces';
+import Alert from '../../common/Alert';
+import { useState } from 'react';
+import {
+  FileDownload,
+  FileDownloadTypeEnum,
+} from '../../../utils/file-download.util';
+import { API_ROUTES } from '../../../constant/apiRoutes';
+import Spinner from '../../common/Spinner';
+
+function PIASubHeader({
+  pia,
+  lastSaveAlertInfo,
+  onSaveChangeClick = () => {},
+  onSubmitClick = () => {},
+}: PIASubHeaderProps) {
+  //TODO implement tooltip for faEllipsisH icon, so when mouse hover, will display download word
+  // this pr just add download function for faEllipsisH icon
+  const [isDownloading, setIsDownloading] = useState(false);
+  const [downloadError, setDownloadError] = useState('');
+  const handleDownload = async () => {
+    setDownloadError('');
+
+    if (!pia.id) {
+      console.error(
+        'Something went wrong. Result Id not available for download',
+      );
+      setDownloadError('Something went wrong. Please try again.');
+      return;
+    }
+
+    setIsDownloading(true);
+    try {
+      await FileDownload.download(
+        API_ROUTES.PIA_INTAKE_RESULT_DOWNLOAD.replace(':id', `${pia.id}`),
+        FileDownloadTypeEnum.PDF,
+      );
+    } catch (e) {
+      setDownloadError('Something went wrong. Please try again.');
+    } finally {
+      setIsDownloading(false);
+    }
+  };
+  const handleAlertClose = () => {
+    setDownloadError('');
+  };
+  return (
+    <>
+      <div className="subheader-container wrapper">
+        <h1>{pia.title ? pia.title : 'New PIA'}</h1>
```

Comment:
Should this be untitled?

Reply:
is this default text "New PIA" explicitly mentioned in the ticket?